course-material/interactive-queries-and-parameterization/introduction-to-ipywidgets.md

```diff
@@ -1,15 +1,376 @@
 ---
-jupyter:
-  jupytext:
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.14.5
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
 ---
 
-# Introduction to ipywidgets
+# Introduction to `ipywidgets`
 
```

Comment:
Rename file from "introduction-to-ipywidgets" to "make-your-queries-interactive"